### PR TITLE
Windows path separator is different from other platform's one

### PR DESCRIPTION
**Pull request by bitfriend**
Will use `path` package to parse path.
Will not split path into filename and extension manually using `/`.

### CODE REVIEW

File: app/lib/common/utils/utils.dart [hunk 1]
```diff
@@ -178,19 +178,19 @@ String getHumanReadableFileSize(int bytes) {
 
 String documentTypeFromFileExtension(String fileExtension) {
   switch (fileExtension) {
-    case 'png':
-    case 'jpg':
-    case 'jpeg':
+    case '.png':
```
Comment (gtalha07) [inline app/lib/common/utils/utils.dart:181]: Wouldn't this require to check if platform is windows? What about other platforms ? Are you sure this wouldn't break the other ones?

**Review by gtalha07 — COMMENTED** [review]
Looks good but I don't have knowledge context whether this'd cover other platforms as well (besides windows). So can't give explicit approval...

File: app/lib/common/utils/utils.dart (hunk 1 above)
Comment (bitfriend) [reply app/lib/common/utils/utils.dart:181]: I think `path` package will support all platforms including windows.

**Comment by bitfriend** [comment]
`path` supports both of windows and other platforms.
Please read [this document](https://pub.dev/packages/path#how-cross-platform-is-this).

**Review by gtalha07 — APPROVED** [review]

**Pull request merged**